src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -212,7 +212,8 @@ public PreSubmitCallbackHandler<AsylumCase> endAppealInternalNotificationHandler
                     return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                             && callback.getEvent() == Event.END_APPEAL
                             && isRepJourney(asylumCase)
-                            && isInternalCase(asylumCase);
+                            && isInternalCase(asylumCase)
+                            && !isAppellantInDetention(asylumCase);
```

Comment:
Added this condition to avoid triggering this method for HomeOffice end appeal. In my situation it is "In Detention"
What do you guys think, is it ok?

Reply:
looks ok yes. Presuming the above is for internal non-detained cases? And you are working on internal ada and non ada right? if all true, then yes should be ok.